### PR TITLE
chore(storybook): bump version

### PR DESCRIPTION
**Pull request by marcelltoth**
Bumps storybook version to 6.2

This actually fixes storybook for me, IDK why but the 6.1 one used to always get into a refresh-loop.

Does not change the builder to Webpack 5 yet, that turns out to be a bit more difficult to do, maybe later in a separate step.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "7.12.1",
+    "@babel/plugin-proposal-object-rest-spread": "7.12.1",
+    "@babel/preset-env": "7.12.11",
+    "@babel/preset-typescript": "7.12.7",
```
Comment (marcelltoth) [inline package.json:21]: This is unfortunately required to work around a bug, see https://github.com/storybookjs/storybook/issues/14197

**Comment by mpodlasin** [comment]
web components don't work (?)

**Comment by mmiask** [comment]
Storybook does not open for me :(

Screenshot 2021-04-22 at 11 42 25

**Comment by mmiask** [comment]
@marcelltoth maybe let's specify the `engine` version range, so we avoid running into errors like this in the future?

```json
"engines": {
    "node": ">=10 <=12.22.1"
  },
```

**Review by mpodlasin — APPROVED** [review]
Works! <3

**Pull request merged**